README.md

```diff
@@ -1,2 +1,49 @@
-# first-capstone-project
-this is the first capstone project for the first module in microverse
+
+# Project Name
+First capstone project in microverse 
+# Project Link
+https://alzubair98.github.io/first-capstone-project/
+
+> Description the project.
+this is the first capstone project where i will build a university portofilo
+
+## Built With
+
+- Major languages
+- Frameworks
+- Technologies used
+
+## Live Demo (if available)
+
+[Live Demo Link](https://alzubair98.github.io/first-capstone-project/)
+
+To get a local copy up and running follow these simple example steps.
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: https://github.com/Alzubair98
+- Twitter: https://twitter.com/FitZubair
+- LinkedIn: https://www.linkedin.com/in/alzubair-alqaraghuli-272918233/
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Alzubair98/Hello---micro/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
```

Comment:
- [ ] Kindly add an acknowledgment to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

Reply:
done